code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -96,13 +100,27 @@
 				update_icon()
 		else
 			to_chat(user, "<span class='warning'>Seems like someone already drew something on this helmet's visor!</span>")
-
+		return
+	if(istype(hitting_item, /obj/item/clothing/head))
+		var/obj/item/clothing/hitting_clothing = hitting_item
+		if(hitting_clothing.clothing_flags & PLASMAMAN_HELMET_EXEMPT)
+			to_chat(user, "<span class='notice'>You cannot place [hitting_clothing.name] on the helmet!</span>")
+			return
+		if(attached_hat)
+			to_chat(user, "<span class='notice'>There's already something placed on the helmet!</span>")
+			return
+		attached_hat = hitting_clothing
+		to_chat(user, "<span class='notice'>You placed [hitting_clothing.name] on the helmet! Keep in mind this is merely cosmetic: hat lights, stats or anything else won't work!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You placed the [hitting_clothing.name] on the helmet! Keep in mind this is merely cosmetic: hat lights, stats or anything else won't work!</span>")
```
should we add the word "the" to here and everywhere else like this?

Reply:
![image](https://user-images.githubusercontent.com/42877449/144312736-ef7aa02f-ec46-4561-b797-e13ab585b795.png)